scio-test/src/test/scala/com/spotify/scio/values/PairHashSCollectionFunctionsTest.scala

```diff
@@ -184,22 +234,21 @@ class PairHashSCollectionFunctionsTest extends PipelineSpec {
     }
   }
 
-  it should "support hashIntersectByKey() with SideSet" in {
+  it should "support hashIntersectByKey() with SideInput[Set[_]]" in {
```

Comment:
```suggestion
  it should "support hashIntersectByKey() with asSetSingletonSideInput" in {
```
To be consistent with other tests?